packages/genesys-spark-components/src/components/stable/gux-popover/gux-popover.tsx

```diff
@@ -215,6 +215,8 @@ export class GuxPopover {
   componentDidLoad(): void {
     if (this.isOpen) {
       this.runUpdatePosition();
+    } else {
```

Comment:
Is this an unrelated change? It should be excluded from this PR

Reply:
Fixed